packages/algorand-js/test/mocks/txn.ts

```diff
@@ -53,7 +56,7 @@ export const TXN_OBJ = {
   apap: Buffer.from("approval"),
   apsu: Buffer.from("clear"),
   apaa: [Buffer.from("arg1"), Buffer.from("arg2")],
-  apat: [Buffer.from("addr-2"), Buffer.from("addr-2")],
+  apat: [Buffer.from(decodeAddress(addr2).publicKey), Buffer.from(decodeAddress(addr2).publicKey)],
```

Comment:
can we update second address to something else ? it will be good for testing.

Reply:
Okay, i will add one more account.